assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -161,31 +161,33 @@ export const actions = {
 		invariant( profileName, 'profileName is required.' );
 		invariant( timezone, 'timezone is required.' );
 
-		let accountTicket;
+		let response,
+			error;
 		yield {
 			payload: {},
```

Comment:
Payload should be the same for all actions except catch which gets the added `error`.

Reply:
Got it!

Reply:
https://github.com/google/site-kit-wp/pull/1390/commits/4b6fa6aedd59bcd4a6f5002d7378eba06276330d